proto/prysm/v1alpha1/BUILD.bazel

```diff
@@ -136,7 +136,8 @@ ssz_electra_objs = [
     "AggregateAttestationAndProofElectra",
     "AttestationElectra",
     "AttesterSlashingElectra",
-    "BeaconBlockElectra",
+    "BeaconBlockBodyElectra",
+    "BeaconBlockContentsElectra",
```

Comment:
why is blinded block contents not needed?

Reply:
This is because (weirdly) https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Validator/produceBlockV3 returns full contents for a non-blinded block and only the block in the blinded case.